src/views/product-landing/helpers/get-icon-cards.tsx

```diff
@@ -10,19 +10,13 @@ import { IconLearn16 } from '@hashicorp/flight-icons/svg-react/learn-16'
 import { ProductData } from 'types/products'
 import { getIsEnabledProductIntegrations } from 'lib/integrations/get-is-enabled-product-integrations'
 
+/**
+ * The order of these items is meaningful, should match the top navigation items
```

Comment:
💡 there could be a more config-driven way to enforce this order across the app, but this felt like an easy first step.

Reply:
https://app.asana.com/0/1204807665183200/1205002760871766/f